On icu-61 built failures look like:
  include/beecrypt/c++/lang/String.h:76:17:
    error: ‘UnicodeString’ does not name a type

ICU-61 moved all names into icu:: space.

The fix is to add 'using icu::<symbol>'.

Reported-by: Juergen Rose
Bug: https://bugs.gentoo.org/666542
--- a/c++/beeyond/DHIESParameterSpec.cxx
+++ b/c++/beeyond/DHIESParameterSpec.cxx
@@ -36,2 +36,4 @@ using std::auto_ptr;
 #include <unicode/numfmt.h>
+using icu::RegexPattern;
+using icu::RegexMatcher;
 
--- a/c++/crypto/Cipher.cxx
+++ b/c++/crypto/Cipher.cxx
@@ -30,2 +30,4 @@ using beecrypt::security::Security;
 #include <unicode/ustream.h>
+using icu::RegexMatcher;
+using icu::RegexPattern;
 
--- a/c++/lang/Integer.cxx
+++ b/c++/lang/Integer.cxx
@@ -29,2 +29,5 @@ using beecrypt::lang::String;
 #include <unicode/numfmt.h>
+using icu::Formattable;
+using icu::NumberFormat;
+
 
--- a/c++/lang/Long.cxx
+++ b/c++/lang/Long.cxx
@@ -29,2 +29,4 @@ using beecrypt::lang::String;
 #include <unicode/numfmt.h>
+using icu::Formattable;
+using icu::NumberFormat;
 
--- a/c++/util/Date.cxx
+++ b/c++/util/Date.cxx
@@ -28,2 +28,3 @@
 #include <unicode/datefmt.h>
+using icu::DateFormat;
 
--- a/include/beecrypt/c++/lang/String.h
+++ b/include/beecrypt/c++/lang/String.h
@@ -41,2 +41,3 @@ using beecrypt::bytearray;
 #include <unicode/unistr.h>
+using icu::UnicodeString;
 
